.github/workflows/releasev2.yml

```diff
@@ -0,0 +1,30 @@
+name: Build Release v2
```

Comment:
Is this need or shall the other release job do all?

Reply:
If you release a v2.x.x tag we build the v2 models if v1.x.x we build the v1?

Should also remove the need for two workflows, since you can trigger on v.X and then add a conditional to the generation steps.